scripts/payload.json

```diff
@@ -4,8 +4,8 @@
     "shape": "tests/testthat/testdata/malawi.geojson",
     "population": "tests/testthat/testdata/population.csv",
     "survey": "tests/testthat/testdata/survey.csv",
-    "programme": "tests/testthat/testdata/programme.csv",
-    "anc": "tests/testthat/testdata/anc.csv"
+    "art_number": "tests/testthat/testdata/programme.csv",
```

Comment:
has this been reflected through the naomi metadata from https://github.com/mrc-ide/naomi/pull/30 ? that might need updating...

Reply:
Ran through this with naomi - the info is using the updated file names as expected